app/controllers/api/v0/signatures_controller.rb

```diff
@@ -19,29 +13,14 @@ def create
 
   def prep_signature_request(manifest)
     signature_request = read_body_as_json(symbolize_names: true)
-    signature_request[:manifest_content] = manifest.content.to_json
-    if signature_request[:token]
-      signature_request[:token] = lookup_signature_token(signature_request[:token])
-    else
-      render json: { message: "Missing signature token" }, status: 403
```

Comment:
I removed this because I think we need to deal with this situation in a more detailed way. Right now, a missing `token` arg returns this message. But if the token is wrong, we return a 422 and different error message. Going to work on improving the error message for both cases.